submodules/LIB/hardware/syn/syn_build.tcl

```diff
@@ -0,0 +1 @@
+set_db hdl_error_on_latch false
```

Comment:
this file should be in iob-soc/hardware/hardware/syn

Not in lib!

Make sure  it is copied to build dir during setup.

Reply:
If the user doesn't want to disable the check, she does not include this content in her syn_build.tcl or doesn't include the file at all.

Reply:
ok implemented. A iob-soc/hardware/syn folder was created. The folder path was already recognized in iob_module  dir_list , so it automaticly copies the syn_build.tl to the buildir/hardware/syn, without having to do changes in iob_module.py.